addons/account/models/account_move.py

```diff
@@ -1718,8 +1718,9 @@ def create(self, vals_list):
 
         vals_list = self._move_autocomplete_invoice_lines_create(vals_list)
         rslt = super(AccountMove, self).create(vals_list)
-        if 'line_ids' in vals_list:
-            rslt.update_lines_tax_exigibility()
+        for vals in vals_list:
+            if 'line_ids' in vals:
+                rslt.update_lines_tax_exigibility()
```

Comment:
Not exactly ;) Here, you're calling the function on all the resulting records, for each dict in vals list, so basically you're doing n² stuff. 
Rather

```suggestion
                for i, vals in enumerate(vals_list):
                    if 'line_ids' in vals:
                        rslt[i].update_lines_tax_exigibility()
```